src/main/java/com/redhat/devtools/intellij/commonuitest/fixtures/mainidewindow/idestatusbar/IdeStatusBar.java

```diff
@@ -65,7 +65,7 @@ public void waitUntilProjectImportIsComplete() {
      * Wait until all the background tasks finish
      */
     public void waitUntilAllBgTasksFinish() {
-        waitFor(Duration.ofSeconds(300), Duration.ofSeconds(10), "The background tasks did not finish in 5 minutes.", this::didAllBgTasksFinish);
+        waitFor(Duration.ofSeconds(600), Duration.ofSeconds(10), "The background tasks did not finish in 10 minutes.", this::didAllBgTasksFinish);
```

Comment:
Well, import in IntelliJ (especially on Windows) takes more time than in Eclipse. In some situations it can takes minutes and nothing is wrong, it is just slow. 300 seconds is failing on windows in one test, maybe double the time for every situation is an overkill. What about to keep 300 as default and give the possibility to set a parameter - custom timeout. I can also try to use 400/500 seconds instead of 600. What do you think?